components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -59,6 +59,8 @@ - (id)copyWithZone:(__unused NSZone *)zone {
   action.accessibilityLabel = self.accessibilityLabel;
   action.titleColor = self.titleColor;
   action.tintColor = self.tintColor;
+  action.dividerColor = self.dividerColor;
+  action.showsDivider = self.showsDivider;
```

Comment:
Should there be updates to the actions in the `setDividerColor` and `setShowsDivider` methods?

Reply:
Do you mean updates in the `MDCActionSheetAction` object when these properties are set?

Reply:
I would suggest that this API should work all the time. I would be surprised if I changed a view controller's property while it was on-screen and it didn't have any effect.  

However, if it's a lot of work to make that happen, then I suppose it's OK to wait until a bug is reported to request it specifically.